### PR TITLE
add documentation for s3 primairy storage

### PR DESCRIPTION
**Pull request by icewind1991**
Requires
- [x] https://github.com/nextcloud/documentation/pull/218
- [x] https://github.com/nextcloud/server/pull/768

### CODE REVIEW

File: admin_manual/configuration_files/primary_storage.rst [hunk 1]
```diff
@@ -68,3 +68,37 @@ The Swift backend mounts a container on an OpenStack Object Storage server into
 			'urlType' => 'internal'
 		),
 	),
+
+--------
+Amazong S3
+--------
+
+The Swift backend mounts a container on an Amazon S3 Storage or compatible server into the virtual filesystem. The class to be used is :code:`\\OC\\Files\\ObjectStore\\S3`
```
Comment (benmichael) [inline admin_manual/configuration_files/primary_storage.rst]: Should this not be
`The S3 backend mounts a bucket on an Amazon S3 Storage`?

Comment (icewind1991) [inline admin_manual/configuration_files/primary_storage.rst]: Good catch, fixed

**Comment by LukasReschke** [comment]
Merge conflict.

**Comment by icewind1991** [comment]
rebased

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**